tests/fast_implementation_test.py

```diff
@@ -55,8 +55,8 @@ def test_ecoli_to_full_dimensional_polytope(self):
 
         steady_states = sampler.generate_steady_states()
 
-        self.assertEqual(sampler.A.shape[0], 175)
-        self.assertEqual(sampler.A.shape[1], 24)
+        self.assertEqual(sampler.A.shape[0], 25)
```

Comment:
great! this shows that the new method generates a smaller model

Reply:
Yes, I agree